From c751866d1efd4346ea0c91476f3b2bb02d6bf90b Mon Sep 17 00:00:00 2001
From: Torge Matthies <openglfreak@googlemail.com>
Date: Sun, 31 Mar 2024 15:49:33 +0200
Subject: [PATCH 10/53] winegstreamer: Connect autoplug-continue and
 deep-element-added callbacks to uridecodebin-based parsers too.

CW-Bug-Id: #21644
---
 dlls/winegstreamer/wg_parser.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/winegstreamer/wg_parser.c b/dlls/winegstreamer/wg_parser.c
index 2b3e19cdab7..8523ef0af1e 100644
--- a/dlls/winegstreamer/wg_parser.c
+++ b/dlls/winegstreamer/wg_parser.c
@@ -2199,6 +2199,7 @@ static BOOL uridecodebin_parser_init_gst(struct wg_parser *parser)
     g_signal_connect(element, "autoplug-select", G_CALLBACK(autoplug_select_cb), parser);
     g_signal_connect(element, "autoplug-sort", G_CALLBACK(autoplug_sort_cb), parser);
     g_signal_connect(element, "no-more-pads", G_CALLBACK(no_more_pads_cb), parser);
+    g_signal_connect(element, "deep-element-added", G_CALLBACK(deep_element_added_cb), parser);
 
     pthread_mutex_lock(&parser->mutex);
     parser->no_more_pads = false;
-- 
2.47.1

